Python/multilateration/multilateration.py

```diff
@@ -14,7 +14,7 @@ def check_invalid_time(
     max_time_difference: float,
 ):
     """Checking if the time difference exceeds the maximum allowed time for a valid signal."""
-    return (abs(sample_diff) * sample_time) > max_time_diff
+    return (abs(sample_diff) * 1 / sample_frequency) > max_time_difference
```

Comment:
@hannahcl I guess this code (and the "parent function" `check_valid_signals`) wasn't used anywhere. because this code should not have been able to run if that was the case.